app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * Configuration for the esbuild bundler and build/watch commands.
+ */
```

Comment:
These docs should be way longer, explaining the whole process and what really happens here. We might understand it now, but this will be the main entry point for someone in the future.

Reply:
I've expanded the comment.